File: HealthCatalyst/ClientApp/app/Models/Patient.ts
import { Description } from './Description';
import { Communication } from './Communication';
import { Image } from './Image';

export class Patient {
    public FirstName: string;
    public LastName: string;
    public Gender: string;
    public DOB: string;
    public Age: number;
    public Communication: Communication;
    public Description: Description;
    public Image: Image;
}

File: HealthCatalyst.Data/DAO/Patient.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace HealthCatalyst.Data.DAO
{
    public class Patient
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Gender { get; set; }
        public string DOB { get; set; }
        public long Age { get; set; }
        public Communication Communication { get; set; }
        public Description Description { get; set; }
        public Image Image { get; set; }
    }
}
